docker/ravendb-ubuntu-arm/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM arm32v7/ubuntu:bionic
+
+RUN apt-get update \
+    && apt-get install -y \
+    && apt-get install --no-install-recommends curl bzip2 libunwind8 libcurl4 ca-certificates jq -y
```

Comment:
If base image is adjusted then this line needs a tweak as well: https://github.com/ravendb/ravendb/blob/v4.2/docker/ravendb-ubuntu/Dockerfile#L5